routes/common/postgres_helper.rb

```diff
@@ -41,6 +41,8 @@ def post(name: nil)
     else 0
     end
 
+    flavor = request_body_params["flavor"] || PostgresResource::Flavor::POSTGRES
```

Comment:
Should we validate this, or does the database-level enum already handle it?

Reply:
Database level enum already handles that. I initially thought intercepting this here to provide a better error message but there is also parallel work going on API level type checking, which I assume handle this as well.